doc/manual/meson.build

```diff
@@ -0,0 +1,75 @@
+if get_option('doc')
+  ignore_headers = [
+    'private.h',
+    'pkcs11.h',
+    'conf.h',
+    'debug.h',
+    'dict.h',
+    'filter.h',
+    'log.h',
+    'mock.h',
+    'modules.h',
+    'pkcs11.h',
+    'pkcs11i.h',
+    'pkcs11x.h',
+    'private.h',
+    'client.h',
+    'proxy.h',
+    'rpc.h',
+    'rpc-message.h',
+    'util.h',
+    'virtual.h',
+    'virtual-fixed.h',
+    'array.h',
+    'compat.h'
+  ]
+
+  content_files = [
+    'p11-kit-config.xml', 'p11-kit-sharing.xml',
```

Comment:
As as suggestion, just to be consistent, wouldn't be better to write one file for each line?